integration-tests/utils/integration-utils.js

```diff
@@ -45,13 +45,10 @@ const getAnimatedDrawer = textContains => {
   return `//*[@role="dialog" and @aria-hidden="false"][contains(normalize-space(.), "${textContains}") or contains(@aria-label,"${textContains}") or @aria-labelledby=//*[contains(normalize-space(.),"${textContains}")]]`
 }
 
-const clickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])`
-  return getClickablePath(base, text, textContains, isDescendant)
-}
-
-const enabledClickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a[@aria-disabled="false"] | //button[@aria-disabled="false"] | //*[@role="button" and @aria-disabled="false"] | //*[@role="link" and @aria-disabled="false"] | //*[@role="combobox" and @aria-disabled="false"] | //*[@role="option" and @aria-disabled="false"])`
+// Note: isEnabled is not supported for native anchor and button elements.
+const clickable = ({ text, textContains, isDescendant = false, isEnabled = true }) => {
+  const checkEnabled = isEnabled !== undefined ? ` and @aria-disabled!="${isEnabled}"` : ''
```

Comment:
This might need a more explicit conditional. If `isEnabled` is `false`, I think the check should be `@aria-disabled="true"`. `@aria-disabled!="false"` will succeed for enabled elements that don't have an `aria-disabled` attribute.

Reply:
I agree and did try that more straightforward implementation. I then got some test failures which indicate that we can't assume aria attributes to always be present on some of these elements-- probably combobox and option. We can either remove the aria check from those elements and see what happens, or we can keep the code as-is and it at least ensures that an element isn't disabled when you expect it to be enabled. I prefer to keep the code as-is, recognizing that there is a weakness here, but it works fine for elements that we always expect to have the aria-disabled property on them.